ruby/example_code/ec2/ec2-ruby-example-create-subnet.rb

```diff
@@ -1,36 +1,113 @@
-# snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
-# snippet-sourceauthor:[Doug-AWS]
-# snippet-sourcedescription:[Creates a public subnet with tags and attaches it to a VPC.]
-# snippet-keyword:[Amazon Elastic Compute Cloud]
-# snippet-keyword:[create_subnet method]
-# snippet-keyword:[Subnet.create_tags method]
-# snippet-keyword:[Ruby]
-# snippet-sourcesyntax:[ruby]
-# snippet-service:[ec2]
-# snippet-keyword:[Code Sample]
-# snippet-sourcetype:[full-example]
-# snippet-sourcedate:[2018-03-16]
-# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX - License - Identifier: Apache - 2.0
+
+require 'aws-sdk-ec2'
+
+# Creates a subnet within a virtual private cloud (VPC) in 
+#   Amazon Virtual Private Cloud (Amazon VPC) and then tags
+#   the subnet.
 #
-# This file is licensed under the Apache License, Version 2.0 (the "License").
-# You may not use this file except in compliance with the License. A copy of the
-# License is located at
+# Prerequisites:
 #
-# http://aws.amazon.com/apache2.0/
+# - A VPC in Amazon VPC.
 #
-# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
-# OF ANY KIND, either express or implied. See the License for the specific
-# language governing permissions and limitations under the License.
+# @param ec2_resource [Aws::EC2::Resource] An initialized
+#   Amazon Elastic Compute Cloud (Amazon EC2) resource object.
+# @param vpc_id [String] The ID of the VPC for the subnet.

```

Comment:
Amazon VPC (everywhere)

Reply:
In this (and other similar) contexts, I'm describing a resource--namely, a virtual private cloud (VPC)--inside of the Amazon VPC service.